composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^7.2",
         "knplabs/knp-components": "^2.0",
         "symfony/config": "^4.3 || ^5.0",
-        "symfony/dependency-injection": "^4.3 || ^5.0",
+        "symfony/dependency-injection": "^4.3 || 5.0.*",
```

Comment:
We just can't.
This would make impossible to install bundle on Symfony 5.1

Reply:
>This would make impossible to install bundle on Symfony 5.1

That's actually the proposition itself.
Failing tests on Symfony 5.1. indicate there is some incompatibility. Restricting Symfony version to 5.0. could be a temporary solution until there will be a patch for bundle compatibility with Symfony 5.1.
Otherwise there will  be constantly failing build and users with Symfony 5.1. may potentially experience problems.

Reply:
Unfortunately, such proposition is unacceptable. We can't restrict someone from using this bundle only because tests are failing, moreover if tests fail because of deprecation messages.
By the way, I'm using this bundle on a Symfony 5.1 project without any problem.